src/main/java/core/basesyntax/service/WriterReportToCsvImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriterReportToCsvImpl implements WriterReportToCsv {
+    private static final String TITLE = "fruit,quantity";
+
+    @Override
+    public void writeReport(Map<String, Integer> storage, String outputFilePath) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(outputFilePath))) {
+            writer.write(TITLE);
+            writer.write(System.lineSeparator());
+            for (Map.Entry<String, Integer> entry : storage.entrySet()) {
+                StringBuilder row = new StringBuilder();
+                row.append(entry.getKey())
+                        .append(",")
+                        .append(entry.getValue());
```

Comment:
Move report generation logic to a separate service. This method should only write content to the file